packages/vaadin-lumo-styles/style.js

```diff
@@ -29,14 +29,14 @@ const style = css`
  * Default values for component-specific custom properties.
  */
 const globals = css`
-  html {
+  * {
```

Comment:
This change is actually not necessary, I believe. This would be needed if we didn't use the internal custom property and override the native property (only relied on the new component-specific property).

Reply:
Thanks, reverted.